cvxpylayers/torch/cvxpylayer.py

```diff
@@ -283,11 +283,18 @@ def forward(ctx, *params):
                 ctx.shapes.append(A.shape)
             info['canon_time'] = time.time() - start
 
-            # compute solution and derivative function
+            # compute solution (always)
+            # and derivative function (if needed for reverse mode)
             start = time.time()
             try:
-                xs, _, _, _, ctx.DT_batch = diffcp.solve_and_derivative_batch(
-                    As, bs, cs, cone_dicts, **solver_args)
+                if any(p.requires_grad for p in params):
+                    xs, _, _, _, ctx.DT_batch = (
+                        diffcp.solve_and_derivative_batch(
+                            As, bs, cs, cone_dicts, **solver_args)
+                    )
+                else:
+                    xs, _, _ = diffcp.cone_program.solve_only_batch(
```

Comment:
Can you open a PR on diffcp to re-export this function from `__init__.py` so we don't need to access it from the `cone_program` namespace?

Reply:
Yup - will do now!

Reply:
Just opened the PR

Reply:
```suggestion
                    xs, _, _ = diffcp.solve_only_batch(
```

Let's use the soon-to-be-newly exported name